pycbc/types/timeseries.py

```diff
@@ -401,6 +401,31 @@ def save_to_wav(self, file_name):
         """
         scaled = _numpy.int16(self.numpy()/max(abs(self)) * 32767)
         write_wav(file_name, self.sample_rate, scaled)
+        
+    def psd(self, segment_duration, avg_method='median'):
```

Comment:
Hmmm... maybe? @titodalcanton was the last to look into this in detail. Want to get that change reviewed? Or do you just want the default to this function be different for now?